vuepress/docs/v6.3.2/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -5,32 +5,36 @@ redirectFrom: /next/tutorials/ecr/tutorials/create-ecr-bundle-from-git.html
 # Build and Publish a Simple Bundle
 
 ## Overview
-This tutorial shows you how to create a simple Entando bundle and deploy it into the Entando Component Repository. This involves manually defining a bundle with a single widget, checking your bundle artifacts into git, apply the Entando bundle custom resource to Kubernetes, and then installing the bundle into your application.
+In this tutorial you will learn how to create a simple Entando bundle and deploy it into the Entando Component Repository. This involves manually defining a bundle with a single widget, checking the bundle artifacts into Git, applying the Entando bundle custom resource to Kubernetes, and then installing the bundle into an application.
 
 ## Prerequisites
-* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) to verify you have the prerequisites in place for this tutorial (e.g. Java, npm, git).
+* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) to verify all dependencies for this tutorial are installed (e.g. Java, npm, Git).
```

Comment:
If it's not too long of a list, would it be a good idea to put the list of dependencies, along with the link to check?

Reply:
by link to check you mean the existing Entando CLI link? i actually think the list is environment-dependent, not too short, and might be subject to change with future revs. Java, npm and Git are for sure required, but i'd hesitate to try for a comprehensive set - that introduces an inconsistent level of detail and there's so much room for error or updates. otherwise i'd agree! @nshaw?

Reply:
Let's hold off on expanding the list for now. That could be in the CLI docs if at all.

Reply:
ideally `ent check-env develop` will print out all the dependencies it checks for and handle missing installations; i have to affirm each request to do so with y